website/project/model.py

```diff
@@ -1451,8 +1453,9 @@ def set_title(self, title, auth, save=False):
         :param str title: The new title.
         :param auth: All the auth information including user, API key.
         """
-        if title is None or not title.strip():
-            raise ValidationValueError('Title cannot be blank.')
+        #Called so validation does not have to wait until save.
+        validate_title(title)
```

Comment:
Otherwise I'd have to add my try/except every time I save a node, instead of every time I just change a title. This allows us to find the validation errors earlier and facilitates error handling and testing.